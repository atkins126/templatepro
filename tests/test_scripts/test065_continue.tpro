{{for cust in customers}}
{{if cust.Name|eq,"Skoda"}}
{{continue}}
{{else}}
{{if cust.Name|eq,"Kia"}}
{{continue}}
{{else}}
{{if cust.Name|eq,"Hyundai"}}
{{continue}}
{{endif}}
{{endif}}
{{endif}}
  Customer: {{:cust.Name}}
{{endfor}}