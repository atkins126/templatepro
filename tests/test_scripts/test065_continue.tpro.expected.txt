  1. Ford
  2. Ferrari
  3. Lotus
  4. FCA
  6. De Tomaso
  7. Dodge
  8. Tesla
  10. Tata
  11. Volkswagen
  12. Audi
